cmd/operator-controller/main.go

```diff
@@ -101,12 +101,14 @@ func main() {
 		cachePath                 string
 		operatorControllerVersion bool
 		systemNamespace           string
-		caCertDir                 string
+		catalogdCertDir           string
+		pullCertDir               string
 		globalPullSecret          string
 	)
 	flag.StringVar(&metricsAddr, "metrics-bind-address", "", "The address for the metrics endpoint. Requires tls-cert and tls-key. (Default: ':8443')")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
-	flag.StringVar(&caCertDir, "ca-certs-dir", "", "The directory of TLS certificate to use for verifying HTTPS connections to the Catalogd and docker-registry web servers.")
+	flag.StringVar(&catalogdCertDir, "catalogd-certs-dir", "", "The directory of TLS certificate to use for verifying HTTPS connections to the Catalogd web service.")
```

Comment:
Why we need change the name?
Why oper-con need `catalogd-certs-dir`?

Reply:
I'm updating the names of the options to better reflect what they are used for, _and_ to make it consistent between operator-controller and catalogd.

operator-controller: 
`--catalogd-certs-dir`: location of CA used to validate connections to catalogd
`--pull-certs-dir`: location of CAs used to validate image pulls

catalogd:
`--pull-certs-dir`: location of CAs used to validate image pulls

These are what the CAs are used for, so the names are clearer. We need to separate these because it's possible they may be different, depending on your deployment.

Reply:
I got thank you 👍